pkg/apis/application/v1alpha1/types.go

```diff
@@ -989,6 +989,11 @@ type ConfigManagementPlugin struct {
 	Generate Command  `json:"generate" protobuf:"bytes,3,name=generate"`
 }
 
+type KustomizeOptions struct {
+	// BuildOptions is a string of build parameters to use
+	BuildOptions string `protobuf:"bytes,1,opt,name=buildOptions"`
```

Comment:
Based on the code changes, I'm not sure what this PR does. Therefore, I would not expect a user to understand. Can we update this with more information AND update `kustomize.md` with an explanation and example?

Reply:
Absolutely